assets/src/domain/eventEditor/ui/tickets/ticketForm/multiStep/useDataListener.ts

```diff
@@ -1,8 +1,14 @@
 import { useEffect } from 'react';
 import { useForm } from 'react-final-form';
+import { pick } from 'ramda';
 
 import { useDataState as useTAMDataState } from '@edtrUI/ticketAssignmentsManager/data';
 import { useDataState as useTPCDataState } from '@edtrUI/tickets/ticketPriceCalculator/data';
+import { Ticket } from '@edtrServices/apollo';
+
+// The fields that need to be synced from TPC to tikcet edit form
+// This is to avoid ticket.name being synced
+const TPC_TICKET_FIELDS_TO_SYNC: Array<keyof Ticket> = ['isTaxable', 'price', 'reverseCalculate'];
```

Comment:
i'm wondering whether it would be better to locate this somewhere within the TPC folder and then import it here?
that could work well if it was part of a file where these fields are handled in TPC, that way if something ever gets added or changed, this constant can be updated at the source.

if there isn't a good spot in TPC to locale this logic, then it's fine as is.

thoughts?

Reply:
There already is a constant in `/domain/eventEditor/ui/tickets/ticketPriceCalculator/utils/constants.ts` named `ticketFieldsToUse` which inlcudes `name` but I deliberately didn't use that to avoid any future issue if we add any other field to TPC. But I think your suggestion is better, I can just remove `name` from  `ticketFieldsToUse` and use it to sync fields. Then in future if other fields are added to that array, we can accordigly update the code here.

Reply:
let's keep things as is then
# 🐑